### DISCUSSION TITLE
Improvement on `Accordion`

### DISCUSSION THREAD

**Discussion by zhondori** [chakra-ui/chakra-ui#8192] [General]
![image](https://github.com/chakra-ui/chakra-ui/assets/83527629/c70ac839-a8a9-4d2b-818e-1b69ad59d3b3)

Even there is no `allowMultiple`, `onChange` callback parameter type is still `ExpandedIndex` which is equal to `number | number[]`.
Can we change it to just `number` if there is no `allowMultiple`?

**Comment by segunadebayo** [comment]
This is leftover v2. `allowMultiple` is gone.

Use `multiple` and `collapsible` on `Accordion.Root`. The change callback then matches that mode.
